templates/shared/_latest_news_strip.html

```diff
@@ -0,0 +1,62 @@
+<noscript>
+  <section class="{% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}">
+    <div class="row">
+      <h3><a href="/blog{% if tag_name %}/tag/{{ tag_name }}{% endif %}">Read the latest {{ heading_topic }} news on our blog&nbsp;&rsaquo;</a></h3>
+    </div>
+  </section>
+</noscript>
+
+<section class="u-hide {% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}" data-js="latest-news">
+  <div class="row">
+    <h3>
+      Latest {{ heading_topic }} news from
+      <a href="/blog{% if tag_name %}/tag/{{ tag_name }}{% endif %}" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'blog', 'eventAction' : 'clicks blog feed link', 'eventLabel' : 'ubuntu.com{% if tag_name %}/{{ tag_name }}{% else %} homepage{% endif %}', 'eventValue' : undefined });">
+        our blog&nbsp;&rsaquo;
+      </a>
+    </h3>
+  </div>
+  <div class="row p-divider">
+    <div class="{% if spotlight %}col-9{% else %}col-12{% endif %}">
+      <div id="horizontal-latest-articles" class="row">
+        <div style="min-height: 9.1rem"><i class="p-icon--spinner u-animation--spin">Loading...</i></div>
+      </div>
+    </div>
+    <div id="spotlight" class="col-3 p-divider__block">
```

Comment:
Could you add `{% if spotlight %}` around this div too to keep things tidy on none spotlight displays